client/src/app/views/form.coffee

```diff
@@ -0,0 +1,37 @@
+$ = require('jquery')
+Marionette = require('backbone.marionette')
+template = require('../templates/form.jade')
+errorMessage = require('../templates/error.jade')
+SiteModel = require('../models/site')
+
+module.exports = Marionette.ItemView.extend(
+  model: new SiteModel()
+  template: template
+  
+  events:
+    'submit form' : 'enterSite'
+  
+  ui:
+    url: '#url'
+    email: '#email'
+  
+  enterSite: (e) ->
+    e.preventDefault()
+    unless @validateForm() then return
+    url = @ui.url.val().trim()
+    email = @ui.email.val().trim()
+    @ui.url.val('')
+    @ui.email.val('')
```

Comment:
Do we want to blank out the input values before the save is successful?  What if we get a server error for instance? Would be nice for users to have their input still there so they can resubmit without having to type it back in.  Maybe only clear the UI in the success callback?